app/models/post.rb

```diff
@@ -0,0 +1,33 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'post_id'
+  has_many :comments, foreign_key: 'post_id'
+
+  after_save :update_post_counter
+
+  validates :title, presence: true
+  validates :text, presence: true
+  validates :likes_counter,
+            numericality: {
+              only_integer: true,
+              greater_than_or_equal_to: 0,
+              message: 'must be an integer greater than or equal to zero'
+            },
+            on: :create
```

Comment:
![image](https://github.com/Munyabelden/Blog_App/assets/93039359/bcc6ab29-64ac-470c-83f7-899cc835f38e)

- [ ] It would be nice if you remove the whole Live Demo section since you don't have any live link 😨 Kindly remove this live demo section, for now, 👍 Once you would have a working live link then you can add this live demo section again with the live link 😊